### PR TITLE
use library.organizationId instead of keep.organizationId

### PR DESCRIPTION
**Pull request by camhashemi**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala [hunk 1]
```diff
@@ -238,14 +238,16 @@ class KifiSiteRouter @Inject() (
...
+
+            library.visibility match {
               case LibraryVisibility.SECRET => request.userIdOpt.exists { userId =>
                 keep.libraryId.exists { libraryId =>
                   libraryMembershipRepo.getWithLibraryId(libraryId).exists(_.userId == userId)
                 }
               }
               case LibraryVisibility.ORGANIZATION => request.userIdOpt.exists { userId =>
-                keep.organizationId.exists { orgId =>
+                library.organizationId.exists { orgId =>
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:250]: Just looking forward, how about `libraries = libraryRepo.getActiveByIds(keep.libraryId.toSet).values.toSet` and `libraries.map(_.organizationId).exists { orgId =>`

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala [hunk 2]
```diff
@@ -238,14 +238,16 @@ class KifiSiteRouter @Inject() (
...
                   orgMembershipRepo.getAllByOrgId(orgId).exists(_.userId == userId)
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:251]: We grab the entire set of org members then loop through them to see if any are the user. I think there's a better way.

Why don't we do `orgMembershipRepo.getByOrgIdsAndUserId(libraries.map(_.organizationId), userId).exists { case (orgId, memOpt) => memOpt.isDefined }`?

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:250]: why's that?

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala (hunk 2 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:251]: Or at the very least, `orgMembershipRepo.getByOrgIdAndUserId(orgId, userId).isDefined`

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:251]: Thanks

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala (hunk 1 above)
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:250]: Works with multiple libraries.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:250]: We may need this at some point, but let's cross that bridge when we get there (we should do a lookup on `LibraryRepo.get` at that time anyways).

**Pull request merged**